cms/templates/widgets/segment-io-footer.html

```diff
@@ -5,10 +5,18 @@
         // We can't use JQuery's on load method because it
         // screws up RequireJS' JQuery initialization.
         var onLoadCallback = function() {
-            analytics.identify("${user.id}", {
-              email: "${user.email}",
-              username: "${user.username}"
-            });
+            analytics.identify(
+                "${user.id}",
+                {
+                    email: "${user.email}",
+                    username: "${user.username}"
+                },
+                {
+                    integrations: {
+                        MailChimp: false
```

Comment:
Is this a temporary solution, or are we planning to keep it around for a while? If it's a long-term solution it might be good to add a small note explaining why we're disabling MailKimp.